internal/adapter/repository/ent/clientdata.go

```diff
@@ -54,6 +56,18 @@ func (r *clientDataRepositoryImpl) DeleteClientData(ctx context.Context, client_
 	return &client_id, nil
 }
 
+func (r *clientDataRepositoryImpl) RevokeAPIKey(ctx context.Context, client_id pulid.ID, apiKey string) (*pulid.ID, string, error) {
```

Comment:
```suggestion
func (r *clientDataRepositoryImpl) UpdateAPIKey(ctx context.Context, client_id pulid.ID, apiKey string) (*pulid.ID, string, error) {
```

usecase層がするのはrevokeだけど、repository層がするのはupdateになる